src/components/Rating.jsx

```diff
@@ -1 +1,24 @@
-// implement Rating component here
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Rating extends Component {
+  render() {
+    const { rating } = this.props;
+
+    return (
+      <div className="rating">
+        {rating}
```

Comment:
Fala meu querido! Ce ta bão? Tava revisando seu código porque já deu pra perceber que tu manja bastante, então eu consigo aprender bastante revisando códigos assim.

Seu código está perfeito, mas gostaria de dar uma dica, não sei se é pertinente ou não...
Na linha 10 você ta colocando o valor de rating, que é um número direto dentro da tag <div>... Será que não seria interessante colocar esse valor dentro de uma outra tag, tipo um p, ou algo assim? 

Um abraço e bons estudos!

```suggestion
        <p>{rating}</p>
```

Reply:
Opa Renan. Ta uma magavilha aqui. Como que andam as coisas por ai? Valeu pelo comentário. Tava me sentindo sozinho já.

Vê, eu concordo contigo para 90% das situação. É sempre bom a gente encapsular nossa informação ao máximo. 

Nesse caso eu tenho apenas o {rating} dentro da div com className="rating", logo eu não tenho de quem isolar ele e essa div já é única e suficiente para eu fazer uma estilização para ele. 

Pode aparecer um caso que criar uma div superior a ela seja interessante para alguma estilização especial.

Fico aguardondo mais code review seu ^^.

Reply:
Tudo tranquilo, o Github não me avisa quando tem notificação então só vi agora a mensagem...

Faz todo sentido o que você disse, a minha preocupação seria mais semântica, porque num aplicação real, supondo que você quisesse futuramente adicionar mais elementos relacionados ao rating do filme, ou conteúdo mesmo, talvez você teria um pouco mais de trabalho por ter que voltar nessa parte do código pra trocar a div ou algo assim.. mas é isso, como eu disse, gosto de ver códigos dos caras bons porque eu sempre aprendo ;) 

Farei isso sim, vou aparecer mais vezes, um grande abraço e bons estudos!